settings.py

```diff
@@ -553,6 +553,11 @@
 CELERY_REMINDER_CASE_UPDATE_QUEUE = CELERY_MAIN_QUEUE
 
 
+# This is the celery queue to use for sending repeat records.
+# It's set to the main queue here and can be overridden to put it
+# on its own queue.
+CELERY_REPEAT_RECORD_QUEUE = CELERY_MAIN_QUEUE
```

Comment:
@benrudolph have you already updated localsettings in all environments to override this? I see you created a worker that consumes from the new queue